packages/cli/src/generateOutputData.js

```diff
@@ -22,7 +22,7 @@ const generateOutputData = (source, params, options) => {
     addMetaData: true
```

Comment:
Let's add a default value for 'parts'

It would be nice to get proper documentation for the parameters as well but that may be asking too much.

Reply:
I must've missed this section. I added the defaults to `packages/cli/src/parseArgs.js`. Resolved in: 1898eaf7230e3df29567c46f3a55231ac8ca4e4e